### PR TITLE
Hometask 2

### PR DESCRIPTION
**Pull request by Saidabdulla**
1. Added new modules (Login, Register)
2. Protected Endpoints
3. Covered endpoints with guards
4. Thank you :)

### CODE REVIEW

File: controllers/authController.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+const users = require("../DB/FakeUsers");
+const bcrypt = require("bcrypt");
+const jwt = require("jsonwebtoken");
+const passport = require("passport");
+const JwtStrategy = require("passport-jwt").Strategy;
+const ExtractJwt = require("passport-jwt").ExtractJwt;
+
+const register = (req, res) => {
+    const user = {
+        id: Math.round(Math.random() * 1000),
```
Comment (Morphine-medicine) [inline controllers/authController.js:10]: Probably need to double check if ID is unique

**Review by Morphine-medicine — APPROVED** [review]
Looks good, just 1 comment, but if I got it right, register is out of scope of this, so that's ok

File: controllers/authController.js (hunk 1 above)
Comment (Saidabdulla) [reply controllers/authController.js:10]: > Probably need to double check if ID is unique

It is just example, I will delete this after using mongodb.

**Comment by Saidabdulla** [comment]
> Looks good, just 1 comment, but if I got it right, register is out of scope of this, so that's ok

Sorry but didn't get, what did you mean by saying "register is out of scope of this"?

**Comment by Saidabdulla** [comment]
> > Looks good, just 1 comment, but if I got it right, register is out of scope of this, so that's ok
> 
> Sorry but didn't get, what did you mean by saying "register is out of scope of this"?

Understand, you mean it is not in MVP. Thanks.

File: controllers/authController.js (hunk 1 above)
Comment (Morphine-medicine) [reply controllers/authController.js:10]: ok, then pr can be merged, good job)

**Pull request merged**